### ISSUE TITLE
Refactor subscriptions as orders

### ISSUE THREAD

**Issue by csjx** [DataONEorg/bookkeeper#65]
Since we don't technically support ongoing `subscriptions` to services, merge them into `orders` throughout the code base.  Also, change `subscribers` to `owners`.  Products are now `purchased` as opposed to `subscribed` to.

**Comment by csjx** [comment]
Subscriptions are now Memberships, and subscribers are owners.  Note that a `Customer.subject` aligns with a `Membership.owner`, and may align with the `Quota.owner` for individuals, but this may be replaced by a group owner `subject`.

**Comment by csjx** [comment]
I was having some difficulties modeling a `Membership` with multiple `products`, and realized that this is not correct.  I discussed this with @gothub, and we realized that a `Subscription` is not actual a `Membership`, but rather an instantiation of a `Product`, tying it to a `Customer` and setting the `Quotas`. I'm reopening this issue, and changing the title so that `Subscriptions` are now `Services` - akin to any online service that somebody purchases (vs subscribes to) and we need to keep track of.
